services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/monitor/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .dynamic_sidecar_api import setup_api_client, shutdown_api_client
```

Comment:
Some notes after renaming:
- `errors.py` I'm fine with it; please note that the director-v2 uses exceptions.py, that's what caused some confusion here
- `monitor` is something that periodically checks the state of dynamic sidecars. If a particular condition in the state happens, it will trigger an event. If this name might not be 100% representative, calling it background I think it is worse. What would you call a monitor which triggers events?

Will rename `monitor` if we find a better name 🙏 help!

Reply:
@pcrespov what about calling it `watcher`, dose it make more sense to you?